zephyr/docker-build.sh

```diff
@@ -15,6 +15,10 @@ unset ZEPHYR_BASE
 # Make sure we're in the right place
 test -e ./sof/scripts/xtensa-build-zephyr.py
 
+# See .github/workflows/zephyr.yml
+PATH="$PATH":/opt/sparse/bin
+command -v sparse  || true
+: REAL_CC="$REAL_CC"
```

Comment:
Please add following code to Zephyr project, file:
(zephyr)/cmake/compiler/gcc/target.cmake:
![image](https://user-images.githubusercontent.com/12482515/190982588-097b05ec-d71e-46c1-9380-e6b929b7d306.png)

Reply:
Thx for the direct link